packages/manager/modules/enterprise-cloud-database/src/create/create.controller.js

```diff
@@ -0,0 +1,10 @@
+export default class EnterpriseCloudDatabaseCreateCtrl {
+  /* @ngInject */
+  constructor(
+    $scope,
+    $http,
+  ) {
+    this.$scope = $scope;
+    this.$http = $http;
+  }
```

Comment:
Why are those needed ? Also wa should prefer going through `ovh-api-services` if possible